app/templates/events/view/sessions/create.hbs

```diff
@@ -1,9 +1,13 @@
 <div class="column sixteen wide">
-  <h2 class="ui header center aligned">
-    {{t 'Create Session'}}
-  </h2>
-  <div class="ui container">
-    {{forms/session-speaker-form fields=model.form data=model save=(action 'save')
-      includeSpeaker=true includeSession=true isSessionSpeaker=true isLoading=isLoading}}
-  </div>
+  {{#if model.tracks}}
```

Comment:
Use `isSessionsSpeakersEnabled`
Implement something like this for both speaker & session pages
![screen shot 2017-10-15 at 9 47 18 pm](https://user-images.githubusercontent.com/12807846/31586728-9e56ba18-b1f2-11e7-94a1-1beba13feafa.png)